datanode/gateway/graphql/schema.graphql

```diff
@@ -599,7 +599,7 @@ type Query {
   ): Order!
 
   "An order in the Vega network found by referenceID"
-  orderByReference("Reference for an order" reference: String!): Order!
+  orderByReference("Reference for an order" reference: String!, filter: OrderFilter): Order!
```

Comment:
Do we actually want/need a filter on this guy?  I would have thought usually you're reference should be pretty unique and you were only expecting one result (though obv. not guaranteed)

Reply:
Yes I think you're right, a bit overzealous with the filtering here, I will remove it.